sqlmesh/core/context.py

```diff
@@ -1426,23 +1425,21 @@ def apply(
         try:
             self._apply(plan, circuit_breaker)
         except Exception as e:
+            tb = traceback.format_exc()
             self.notification_target_manager.notify(
                 NotificationEvent.APPLY_FAILURE,
                 environment=plan.environment_naming_info.name,
                 plan_id=plan.plan_id,
-                exc=traceback.format_exc(),
+                exc=tb,
             )
-            self.log_failure_error(method="Apply")
+            logger.info("Plan application failed.\n" + tb)
```

Comment:
Can't we just use native logging capabilities to log this? Like `logger.inf(..., exc_info=e)`?